contracts/src/main/kotlin/com/r3/developers/csdetemplate/contracts/TestConsensualState.kt

```diff
@@ -0,0 +1,12 @@
+package com.r3.developers.csdetemplate.contracts
+
+import net.corda.v5.ledger.consensual.ConsensualState
+import net.corda.v5.ledger.consensual.transaction.ConsensualLedgerTransaction
+import java.security.PublicKey
+
+class TestConsensualState(
+    val testField: String,
+    override val participants: List<PublicKey>
+) : ConsensualState {
+    override fun verify(ledgerTransaction: ConsensualLedgerTransaction) {}
+}
```

Comment:
Don't we use some mocking infrastructure to the interfaces/abstract classes that avoid the definition of such classes as TestConsensualState ?